letter-shop/index.md

```diff
@@ -0,0 +1,634 @@
+
+```js
+console.log(` 1. Connecting to an account to accept payments...`)
+
+plugin.connect().then(function () {
+  
+  //Get ledger and account information from the plugin
+  const ledgerInfo = plugin.getInfo()
+  const account = plugin.getAccount()
+
+  console.log(`    - Connected to ledger: ${ledgerInfo.prefix}`)
+  console.log(`    -- Account: ${account}`)
+  console.log(`    -- Currency: ${ledgerInfo.currencyCode}`)
+  console.log(`    -- CurrencyScale: ${ledgerInfo.currencyScale}`)
+
+  //Covert our cost (10) into the right format given the ledger scale
+  const normalizedCost = cost / Math.pow(10, parseInt(ledgerInfo.currencyScale))
+
+  // Handle incoming web requests...
+
+  // Handle incoming transfers...
+
+})
+```
+
+Stop the server and restart it with the new code. Now it will use the ledger plugin to connect to the XRP Testnet Ledger. When it is connected (usually a few seconds) you'll see some logging in the console providing info about the ledger.
+
+We can get a lot of info from the plugin about the ledger and the account it is connected to using the `getInfo()` and `getAccount()` methods.
+
+> **NOTE:** The currency of the ledger is XRP and the currency scale is 6. That means that to send 1 XRP to this ledger we must send an Interledger payment with an amount of 1000000 (1 million Drops).
+
+> **Interledger standardizes on 64-bit unsigned integers for amounts.**
+
+> Note how we normalize the cost of our service (10 Drops) to be able to express it in XRP (0.00001 XRP) for display purposes.
+
+> **IMPORTANT:** Scale (and precision) can be a confusing aspect of any financial protocol and you'd do well to understand the ledger you are working with and be sure that when you are sending payments and displaying amounts to users you are getting your scale and precision right. We made the decision to avoid some complexity by using only scale and requiring that ledgers always express their currency with a precision of zero. This allowed us to only use integers (not decimals) in the protocol so all numbers are normalised to that form. There are a lot of other technical reasons behind the decision which you'll find discussed at length in the project issue list if you are interested.
+
+Now we have the plugin connected and we have the info we need about the account where we will be accepting payments. 
+
+:tada: Congratulations, our shop is performing step 1 of our 5 step plan!
+
+Our next job is to start up the web server that will host our Letter Shop service.
+
+Whenever a customer visits the website we'll try to find a proof-of-payment token in the URL. If the token is there we'll try to redeem it and give the customer their letter. If not we'll generate the letter and the proof-of-payment token and store these for later. We'll then generate another token that is derived from the proof-of-payment token and give this to the customer to attach to their payment so we can reconcile the payment and this original request.
```

Comment:
The "We'll then generate another token" part becomes a bit too much, I think. Maybe just keep it at "If not we'll generate the letter and the payment parameters, and store these for later."

Reply:
I think that's an important aspect of the process (getting the condition). Will reword it to make it easier to digest.